### DISCUSSION TITLE
Generate G-Code instead of Print option as default?

### DISCUSSION THREAD

**Discussion by JendaDH** [OrcaSlicer/OrcaSlicer#935] [Q&A]
Hi, 

is there a way how to chose Export G-Code as default option after slicing instead of Print option?
Thanks,

JendaDH

**Comment by chibiconsulting** [comment]
Any idea if this could be implemented?
Example:
Preferences > Output button default > print|save G-code
also for comleteness:
Preferences > Slice button default > Slice all|plate
